src/Carbon/Lang/lv.php

```diff
@@ -31,72 +32,111 @@
+        'dienas' => 'dienām',
+        'dienu' => 'dienām',
+        'stunda' => 'stundas',
+        'stundas' => 'stundām',
+        'stundu' => 'stundām',
+        'minūte' => 'minūtes',
+        'minūtes' => 'minūtēm',
+        'minūšu' => 'minūtēm',
+        'sekunde' => 'sekundes',
+        'sekundes' => 'sekundēm',
+        'sekunžu' => 'sekundēm',
+    ]);
+};
+
 return [
-    'year' => '0 gadiem|:count gads|:count gadiem',
-    'y' => '0 gadiem|:count gads|:count gadiem',
-    'a_year' => '{1}gads|0 gadiem|:count gads|:count gadiem',
-    'month' => '0 mēnešiem|:count mēneša|:count mēnešiem',
+    'ago' => function ($time) use ($transformDiff) {
+        return 'pirms '.$transformDiff($time);
+    },
+    'from_now' => function ($time) use ($transformDiff) {
+        return 'pēc '.$transformDiff($time);
+    },
+
+    'year' => '0 gadu|:count gads|:count gadi',
+    'y' => ':count g.',
+    'a_year' => '{1} gads|0 gadu|:count gads|:count gadi',
+    'month' => '0 mēnešu|:count mēnesis|:count mēneši',
     'm' => ':count mēn.',
-    'a_month' => '{1}mēneša|0 mēnešiem|:count mēneša|:count mēnešiem',
-    'week' => '0 nedēļām|:count nedēļas|:count nedēļām',
+    'a_month' => '{1} mēnesis|0 mēnešu|:count mēnesis|:count mēneši',
+    'week' => '0 nedēļu|:count nedēļa|:count nedēļas',
     'w' => ':count ned.',
-    'a_week' => '{1}nedēļas|0 nedēļām|:count nedēļas|:count nedēļām',
-    'day' => '0 dienām|:count dienas|:count dienām',
-    'd' => '0 dienām|:count dienas|:count dienām',
-    'a_day' => '{1}dienas|0 dienām|:count dienas|:count dienām',
-    'hour' => '0 stundām|:count stundas|:count stundām',
+    'a_week' => '{1} nedēļa|0 nedēļu|:count nedēļa|:count nedēļas',
+    'day' => '0 dienu|:count diena|:count dienas',
+    'd' => ':count d.',
+    'a_day' => '{1} diena|0 dienu|:count diena|:count dienas',
+    'hour' => '0 stundu|:count stunda|:count stundas',
     'h' => ':count st.',
-    'a_hour' => '{1}stundas|0 stundām|:count stundas|:count stundām',
-    'minute' => '0 minūtēm|:count minūtes|:count minūtēm',
+    'a_hour' => '{1} stunda|0 stundu|:count stunda|:count stundas',
+    'minute' => '0 minūšu|:count minūte|:count minūtes',
     'min' => ':count min.',
-    'a_minute' => '{1}minūtes|0 minūtēm|:count minūtes|:count minūtēm',
-    'second' => '0 sekundēm|:count sekundes|:count sekundēm',
+    'a_minute' => '{1} minūte|0 minūšu|:count minūte|:count minūtes',
+    'second' => '0 sekunžu|:count sekunde|:count sekundes',
     's' => ':count sek.',
-    'a_second' => '{1}dažas sekundes|0 sekundēm|:count sekundes|:count sekundēm',
-
-    'ago' => 'pirms :time',
-    'from_now' => 'pēc :time',
+    'a_second' => '{1} sekunde|0 sekunžu|:count sekunde|:count sekundes',
 
     'after' => ':time vēlāk',
-    'before' => ':time pirms',
-
     'year_after' => '0 gadus|:count gadu|:count gadus',
-    'a_year_after' => '{1}gadu|0 gadus|:count gadu|:count gadus',
+    'a_year_after' => '{1} gadu|0 gadus|:count gadu|:count gadus',
```

Comment:
Can't tell if the space should be here or not, because there are no tests for this and I don't know how to make Carbon output this.